src/app/components/modules/UserWallet.jsx

```diff
@@ -574,6 +576,18 @@ class UserWallet extends React.Component {
                 <div className="UserWallet__balance row zebra">
                     <div className="column small-12 medium-8">
                         STEEM POWER
+                        <HelpTip
+                            content={[
+                                <h4 key="sp-helptip">What is Steem Power?</h4>,
+                                <p key="sp-helptip-1">
+                                    'Steem Power - Steem Power (abbreviated SP)
+                                    is a measurement of how much influence a
+                                    user has in the Steem network.'
+                                </p>,
+                            ]}
+                        >
+                            <IconButton size="small" fill="green" />
```

Comment:
the icon shouldn't be a pencil, it should be a question mark as speced in https://github.com/steemit/condenser/issues/2540

Reply:
Ok I have done thate.